lib/streamlit/elements/markdown.py

```diff
@@ -17,6 +17,7 @@
 from streamlit import type_util
 from streamlit.proto.Markdown_pb2 import Markdown as MarkdownProto
 from .utils import clean_text
+from ..type_util import SupportsStr
```

Comment:
heads up that this will require a small change as I recently got rid of `elements.utils.type_util` in favor of an identical function in `string_util` in #5121

Reply:
Thanks for the heads up :)